Dockerfile.in

```diff
@@ -1,11 +1,16 @@
 FROM alpine/helm:VERSION
 ENV HOME /tmp
 
+#Install yq and jq dependencies
+RUN apk update && \
+    apk add py-pip jq && \
+    pip install yq
+
 #FIXME: add versioning to helm-s3
 
-RUN helm init -c && \
-    apk add --update --no-cache git bash && \
+RUN apk add --update --no-cache git bash && \
     helm plugin install https://github.com/hypnoglow/helm-s3.git && \
+    helm plugin install https://github.com/chartmuseum/helm-push.git && \
```

Comment:
Since this is only used for one chart, it would be best to install it while building that chart instead of baking it in the image.

Reply:
This plugin is needed to push all the charts to ChartMuseum specified in version.yaml, so i am not really sure what you mean by "only used for one chart" ?

Reply:
Oh, I thought it was for pulling the chart museum chart. Disregard.